runelite-client/src/main/java/net/runelite/client/util/SwingUtil.java

```diff
@@ -186,20 +183,6 @@ public void mouseClicked(MouseEvent e)
 		return trayIcon;
 	}
 
-	/**
-	 * Check if point is in screen bounds.
-	 *
-	 * @param x the x
-	 * @param y the y
-	 * @return the boolean
-	 */
-	public static boolean isInScreenBounds(final int x, final int y)
```

Comment:
As I understand it, the changes in this file were from a cherry-picked commit from the "Remove custom UI" PR, but I'd like to see them be removed or in a separate commit since they aren't used in the substantive part of this change.